build/deploy/db_schemas/defaultdb/000003_migrate_data_to_index.up.sql

```diff
@@ -26,4 +26,11 @@ FROM compact_sub_cells
 WHERE subs.id = compact_sub_cells.subscription_id
     AND cells IS NULL;
 
-COMMIT;
\ No newline at end of file
+COMMIT;
+
+ALTER TABLE identification_service_areas ALTER COLUMN cells SET NOT NULL;
```

Comment:
Do these statements not need a trailing COMMIT?

Reply:
No, they will not need a COMMIT statement as they are not moving any data around and therefore can be done outside of a transaction.